src/olgm/listener/PropertyListener.js

```diff
@@ -17,7 +17,11 @@ class PropertyListener extends Listener {
   constructor(target, oldTarget, key, listen) {
     super(target.on('change:' + key, e => {
       if (this.innerListener_) {
-        this.innerListener_.unlisten();
+        if (Array.isArray(this.innerListener_)) {
+          this.innerListener_.forEach(listener => listener.unlisten());
```

Comment:
Let's empty the array after unlistening the events, i.e. let's add:

```
this.innerListener_.length = 0;
```

Reply:
A few lines below there is assigned a new object for this.innerListener_. IMHO, it is not needed.
`this.innerListener_ = listen(e.target.get(e.key), e.oldValue);`

> Let's empty the array after unlistening the events, i.e. let's add:
> 
> ```
> this.innerListener_.length = 0;
> ```